server/core/hot_region_storage.go

```diff
@@ -294,8 +295,11 @@ func (h *HotRegionStorage) delete() error {
 	db := h.LeveldbKV
 	batch := new(leveldb.Batch)
 	for _, hotRegionType := range HotRegionTypes {
+		if h.hotRegionStorageHandler.GetHotRegionsReservedDaysConfig() == 0 {
```

Comment:
Maybe use a variable to replace h.hotRegionStorageHandler.GetHotRegionsReservedDaysConfig()